_i18n/en/resources/user-guides/monerod-as-a-systemd-service.md

```diff
@@ -0,0 +1,73 @@
+# Using monerod as a systemd Service
+
+If you are using a Linux distribution that uses `systemd`, you can use it's
+user units to manage `monerod` as well.
+
+We will do this in the command line & assume that you have already built or
+downloaded `monerod`.
+
+
+## Create the Service File
+
+First start by creating the directory for your user service:
+
+    mkdir -p ~/.config/systemd/user/
+
+Then create `~/.config/systemd/user/monerod.service` with the following content:
+
+    [Unit]
```

Comment:
Please use the suggest unit file from https://github.com/monero-project/monero/blob/master/utils/systemd/monerod.service

Make sure to use `PrivateTmp=true`.